debian/postinst

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+/usr/lib/dracut/dracut-install --all --fips --ldd -D /usr/lib/ubuntu-core-initramfs/fips /usr/bin/sha512hmac || true
```

Comment:
@alfonsosanchezbeato @valentindavid is this how we envision gathering skeleton at runtime, from packages that ubuntu-core-initramfs depends on? Or do you see this being listed more and more inside ubuntu-core-initramfs script?

Reply:
We are already taking snapd bits when `ubuntu-core-initramfs` is run, so to be consistent this should be done from there as well. Imo doing things from there makes things more dynamic for developers building the initramfs from a container and avoids errors if you forget to reinstall the package. Not sure if there is a particular advantage when doing this from maintainer scripts.